package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "react-spectre",
   "version": "0.0.1",
-  "description": "React components for Spectre.css - a Lightweight, Responsive and Modern CSS Framework.",
+  "description":
+    "React components for Spectre.css - a Lightweight, Responsive and Modern CSS Framework.",
```

Comment:
Why the new line? NPM/Yarn will reformat this.

Reply:
I think this was my editor fault